src/app/cli/src/coda_worker_testnet.ml

```diff
@@ -55,29 +55,27 @@ module Api = struct
     Coda_process.disconnect (List.nth_exn t.workers i)
 
   let send_payment t i sender_sk receiver_pk amount fee =
+    let open Deferred.Option.Let_syntax in
```

Comment:
With this, would we need the function `send_payment_with_receipt`? If not, please replace the usages of `send_payment_with_receipt` with this function.

Reply:
I don't think we need this function. I would delete this.